themes/app/src/js/components/CustomNav.vue

```diff
@@ -2,16 +2,18 @@
     <nav class="main-nav">
         <div class="container">
             <div class="main-nav__top-menu row justify-content-between">
-                <a class="main-nav__logo col-md-6 col-6" href="/"><img/></a>
+                <a class="col-md-6 col-6" href="/">
```

Comment:
Avoid using `/` and use `$BaseHref` instead :)

Reply:
Will do now 👌🏻